minikube-operator-example/cmd/manager/platform.go

```diff
@@ -26,7 +26,11 @@ import (
 	"knative.dev/serving-operator/pkg/reconciler/knativeserving/common"
 )
 
-var log *zap.SugaredLogger
+var (
+	log *zap.SugaredLogger
+	// Platform contains the Minikube-specific tranformer functions
+	platform = common.Platforms{Configure}
```

Comment:
what are your thoughts on calling this variable minikube? 

That way in main it's clear what is minikube vs system

Reply:
Agree. Will change it to `minikubePlatform`